eth/go.mod

```diff
@@ -3,7 +3,7 @@ module pkg.berachain.dev/polaris/eth
 go 1.20
 
 // Required for stateful precompiles and supporting the Ethereum JSON-RPC API.
-replace github.com/ethereum/go-ethereum => github.com/berachain/polaris-geth v0.0.0-20230313222742-b18b15fecd70
+replace github.com/ethereum/go-ethereum => github.com/berachain/polaris-geth v0.0.0-20230321144626-29251e594538
```

Comment:
you can add this to the cosmos/ and playground/ go.mod's as well

Reply:
done